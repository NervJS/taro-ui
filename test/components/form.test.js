import Nerv from 'nervjs'
import { renderToString } from 'nerv-server'
import AtForm from '../../.temp/components/form/index'

describe('AtForm Snap', () => {
  it('render initial AtForm', () => {
    const component = renderToString(<AtForm />)
    expect(component).toMatchSnapshot()
  })

  it('render AtForm -- props reportSubmit', () => {
    const component = renderToString(<AtForm reportSubmit />)
    expect(component).toMatchSnapshot()
  })
})
