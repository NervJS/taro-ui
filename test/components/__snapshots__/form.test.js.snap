// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AtForm Snap render AtForm -- props reportSubmit 1`] = `"<form class=\\"at-form\\"></form>"`;

exports[`AtForm Snap render initial AtForm 1`] = `"<form class=\\"at-form\\"></form>"`;
